controller/rest/bench.go

```diff
@@ -49,7 +49,7 @@ func bench2REST(bench share.BenchType, item *share.CLUSBenchItem, cpf *complianc
 			}
 		} else {
 			if _, ok := cpf.filter[r.TestNum]; ok {
-				r.Tags = metaMap[r.TestNum].Tags
+				r.Tags = cpf.filter[r.TestNum]
```

Comment:
why not just
```
                        if tags, ok := cpf.filter[r.TestNum]; ok {
				r.Tags = tags
			}
```
? Same for lines 37/39 & 90/91 (like in lines 83/85)

Reply:
you have a point